data.json

```diff
@@ -1,4 +1,10 @@
 [
+  {
```

Comment:
Please add yourself to the bottom :nerd_face:

Reply:
Okay!